noetic/distribution.yaml

```diff
@@ -8325,7 +8325,7 @@ repositories:
       tags:
         release: release/noetic/{package}/{version}
       url: https://github.com/septentrio-users/septentrio_gnss_driver-release.git
-      version: 1.0.6-1
+      version: 1.0.8-1
```

Comment:
Same comment as the melodic PR https://github.com/ros/rosdistro/pull/31046/files#r739585631. I don't see 1.0.8 in the release repo


https://github.com/septentrio-users/septentrio_gnss_driver-release/tree/master